--- lib/Text/DHCPLeases/Object.pm	2008-06-05 12:31:04.000000000 -0400
+++ /usr/lib/perl5/site_perl/5.8.8/Text/DHCPLeases/Object.pm	2008-07-01 15:25:40.000000000 -0400
@@ -178,7 +178,7 @@
 	    $obj{'next_binding_state'} = $1;
 	}elsif ( /binding state (\w+);/ ){
 	    $obj{'binding_state'} = $1;
-	}elsif ( /uid (\".*\");/ ){
+	}elsif ( /uid (.*);/ ){
 	    $obj{uid} = $1;
 	}elsif ( /client-hostname (\".*\");/ ){
 	    $obj{'client_hostname'} = $1;
